content/docs/06-integrations/00-canonical-k8s.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Canonical Micro Kubernetes"
+metaTitle: "Canonical Micro Kubernetes"
+metaDescription: "Canonical Micro Kubernetes pack in Spectro Cloud"
+hiddenFromNav: true
+isIntegration: true
+category: ["kubernetes"]
+logoUrl: "https://registry.dev.spectrocloud.com/v1/kubernetes-microk8s/blobs/sha256:b971b64f62e2e67b0a166316f96e6f4211aacea6e28459bb89275e8882ade985?type=image/png"
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+import PointsOfInterest from 'shared/components/common/PointOfInterest';
+import Tooltip from "shared/components/ui/Tooltip";
+
+# Canonical Micro Kubernetes
+
+
+MicroK8s is a scalable Kubernetes distribution providing core Kubernetes functionalities with optimal resource consumption. 
+
+With limited resource requirements, MicroK8s extends its support in:
+
+* Turning Kubernetes into a lightweight development tool.
+
+
+* Making Kubernetes available for use in minimal environments such as GitHub CI.
+
+
+* Adapting Kubernetes for small-appliance IoT applications.
+
+# Prerequisites
+
+* An Ubuntu Operating System
+
+
+* 20GB Storage
+
+
+* 4G Memory
+
```

Comment:
The above "prerequisites" seem to come from https://microk8s.io/docs/getting-started. Let's make a reference to their site if reader wants more info.

Reply:
in references, section added the site